ngraph/core/src/op/multiply.cpp

```diff
@@ -41,6 +41,10 @@ bool evaluate_multiply(const HostTensorPtr& arg0,
         NGRAPH_TYPE_CASE(evaluate_multiply, f16, arg0, arg1, out, broadcast_spec);
         NGRAPH_TYPE_CASE(evaluate_multiply, f32, arg0, arg1, out, broadcast_spec);
         NGRAPH_TYPE_CASE(evaluate_multiply, bf16, arg0, arg1, out, broadcast_spec);
+        NGRAPH_TYPE_CASE(evaluate_multiply, i8, arg0, arg1, out, broadcast_spec);
```

Comment:
Do we have real cases which require these data types?

Reply:
As was discussed with author @antonvor,  there are no such cases. Removed.